Sync 'html/dom/elements/global-attributes' from WPT upstream

Ahmad-S792 opened the pull request:
#### 268bc370a22870886301f87d4121d9d6e1d36708
<pre>
Sync &apos;html/dom/elements/global-attributes&apos; from WPT upstream

<a href="https://bugs.webkit.org/show_bug.cgi?id=267962">https://bugs.webkit.org/show_bug.cgi?id=267962</a>

Reviewed by Anne van Kesteren.

This patch is to sync &apos;html/dom/elements/global-attributes&apos; WPT tests from upstream:

Upstream commit: <a href="https://github.com/web-platform-tests/wpt/commit/0e090696ed5db67de5f50256f4a6abe59733632a">https://github.com/web-platform-tests/wpt/commit/0e090696ed5db67de5f50256f4a6abe59733632a</a>

&gt; Files Changed:
* LayoutTests/imported/w3c/resources/resource-files.json:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-auto-form-associated.window-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-auto-form-associated.window.js:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir_auto-N-EN-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir_auto-N-EN-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir_auto-N-EN.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir_auto-textarea-N-between-Rs.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/lang-xmllang-01-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/lang-xmllang-01-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/lang-xmllang-01.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/lang-xyzzy.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/w3c-import.log:

&gt; Deleted Files:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-slots-directionality.tentative-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-slots-directionality.tentative.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir_auto-N-EN-ref-expected.html:

&gt; New Tests &amp; Expectations:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-assorted.window-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-assorted.window.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-assorted.window.js:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-auto-div-append-child-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-auto-div-append-child.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-auto-dynamic-changes.window-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-auto-dynamic-changes.window.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-auto-dynamic-changes.window.js:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-01-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-01-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-01.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-02-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-02-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-02.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-03-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-03-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-03.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-04-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-04-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-04.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-05-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-05-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-05.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-06-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-06-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-06.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-07-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-07-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-07.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-08-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-08-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-08.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-09-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-09-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-09.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-10-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-10-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-10.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-11-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-11-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-11.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-12-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-12-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-12.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-13-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-13-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-13.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-14-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-14-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-14.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-15-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-15-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-15.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-16-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-16-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-16.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-17-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-17-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-17.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-18-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-18-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-18.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-19-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-19-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-19.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-20-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-20-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-20.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-21-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-21-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-21.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-22-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-22-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-22.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-23-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-23-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-23.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-24-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-24-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-24.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-25-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-25-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-25.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-26-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-26-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-26.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-27-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-27-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-27.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-28-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-28-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-28.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-29-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-29-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-29.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-30-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-30-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-30.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-31-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-31-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-31.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-32-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-32-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-32.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-33-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-33-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-33.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-34-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-34-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-34.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-35-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-35-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-35.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-36-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-36-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-36.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-37-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-37-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-37.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-38-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-38-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-38.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-39-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-39-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-39.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-40-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-40-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-40.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-41-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-41-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-41.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-utils.js:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-slots-directionality-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-slots-directionality.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/lang-attribute-shadow.window-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/lang-attribute-shadow.window.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/lang-attribute-shadow.window.js:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/lang-attribute.window-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/lang-attribute.window.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/lang-attribute.window.js:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/the-anchor-attribute-001.tentative-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/the-anchor-attribute-001.tentative.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/the-anchor-attribute-002.tentative-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/the-anchor-attribute-002.tentative.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/the-anchor-attribute-003-crash.tentative.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/the-anchor-attribute-003.tentative-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/the-anchor-attribute-003.tentative.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/translate-non-html-translation-mode-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/translate-non-html-translation-mode.html:

&gt; Test Expectation Update:
* LayoutTests/TestExpectations: Added as [ImageOnly Failure]
* LayoutTests/platform/mac/TestExpectations: Add ‘asserting’ test on Debug as ‘skip’
* LayoutTests/platform/mac-wk1/mported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-auto-form-associated.window-expected.txt: Platform Specific Test Update
* LayoutTests/platform/wpe/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-auto-form-associated.window-expected.txt: Platform Specific Test Update

Canonical link: <a href="https://commits.webkit.org/273493@main">https://commits.webkit.org/273493@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/b56f0a7911ce2870c4e177cc14d06cdfadc26bd4

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/35524 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/48/builds/14471 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/14/builds/37660 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/38264 "Built successfully") | [✅ 🛠 wincairo](https://ews-build.webkit.org/#/builders/32/builds/32027 "Built successfully") 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/49/builds/16850 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/51/builds/11501 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/30840 "Passed tests") | 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/36076 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/47/builds/12223 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/31628 "Passed tests") | [✅ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/10725 "Passed tests") | 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/42/builds/10757 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/10/builds/31762 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/39510 "Built successfully") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/32286 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/36/builds/32092 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/36716 "Passed tests") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/44/builds/10938 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/50/builds/8835 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/34769 "Passed tests") | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/46/builds/12655 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/8/builds/31417 "Passed tests") | | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/19/builds/8126 "Built successfully and passed tests") | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/43/builds/11454 "Built successfully") | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/45/builds/11718 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

annevk reviewed (approved):
Thanks for doing this. Looks good assuming the bots are happy. If we already have a bug tracking those dir failures maybe that should be mentioned in TestExpectations.

Ahmad-S792 committed:
Sync 'html/dom/elements/global-attributes' from WPT upstream

https://bugs.webkit.org/show_bug.cgi?id=267962

Reviewed by Anne van Kesteren.

This patch is to sync 'html/dom/elements/global-attributes' WPT tests from upstream:

Upstream commit: https://github.com/web-platform-tests/wpt/commit/0e090696ed5db67de5f50256f4a6abe59733632a

> Files Changed:
* LayoutTests/imported/w3c/resources/resource-files.json:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-auto-form-associated.window-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-auto-form-associated.window.js:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir_auto-N-EN-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir_auto-N-EN-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir_auto-N-EN.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir_auto-textarea-N-between-Rs.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/lang-xmllang-01-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/lang-xmllang-01-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/lang-xmllang-01.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/lang-xyzzy.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/w3c-import.log:

> Deleted Files:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-slots-directionality.tentative-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-slots-directionality.tentative.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir_auto-N-EN-ref-expected.html:

> New Tests & Expectations:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-assorted.window-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-assorted.window.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-assorted.window.js:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-auto-div-append-child-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-auto-div-append-child.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-auto-dynamic-changes.window-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-auto-dynamic-changes.window.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-auto-dynamic-changes.window.js:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-01-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-01-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-01.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-02-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-02-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-02.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-03-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-03-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-03.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-04-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-04-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-04.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-05-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-05-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-05.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-06-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-06-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-06.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-07-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-07-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-07.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-08-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-08-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-08.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-09-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-09-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-09.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-10-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-10-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-10.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-11-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-11-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-11.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-12-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-12-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-12.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-13-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-13-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-13.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-14-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-14-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-14.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-15-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-15-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-15.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-16-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-16-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-16.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-17-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-17-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-17.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-18-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-18-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-18.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-19-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-19-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-19.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-20-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-20-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-20.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-21-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-21-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-21.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-22-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-22-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-22.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-23-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-23-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-23.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-24-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-24-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-24.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-25-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-25-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-25.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-26-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-26-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-26.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-27-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-27-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-27.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-28-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-28-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-28.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-29-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-29-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-29.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-30-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-30-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-30.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-31-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-31-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-31.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-32-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-32-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-32.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-33-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-33-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-33.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-34-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-34-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-34.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-35-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-35-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-35.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-36-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-36-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-36.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-37-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-37-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-37.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-38-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-38-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-38.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-39-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-39-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-39.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-40-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-40-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-40.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-41-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-41-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-41.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-shadow-utils.js:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-slots-directionality-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-slots-directionality.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/lang-attribute-shadow.window-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/lang-attribute-shadow.window.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/lang-attribute-shadow.window.js:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/lang-attribute.window-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/lang-attribute.window.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/lang-attribute.window.js:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/the-anchor-attribute-001.tentative-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/the-anchor-attribute-001.tentative.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/the-anchor-attribute-002.tentative-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/the-anchor-attribute-002.tentative.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/the-anchor-attribute-003-crash.tentative.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/the-anchor-attribute-003.tentative-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/the-anchor-attribute-003.tentative.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/translate-non-html-translation-mode-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/translate-non-html-translation-mode.html:

> Test Expectation Update:
* LayoutTests/TestExpectations: Added as [ImageOnly Failure]
* LayoutTests/platform/mac/TestExpectations: Add ‘asserting’ test on Debug as ‘skip’
* LayoutTests/platform/mac-wk1/mported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-auto-form-associated.window-expected.txt: Platform Specific Test Update
* LayoutTests/platform/wpe/imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir-auto-form-associated.window-expected.txt: Platform Specific Test Update

Canonical link: https://commits.webkit.org/273493@main
